src/Components/CreateImageWizardV2/ValidatedTextInput.tsx

```diff
@@ -7,16 +7,105 @@ import {
   TextInputProps,
 } from '@patternfly/react-core';
 
+import { useAppDispatch, useAppSelector } from '../../store/hooks';
+import {
+  setStepInputValidation,
+  selectInputValidation,
+} from '../../store/wizardSlice';
+
 interface ValidatedTextInputPropTypes extends TextInputProps {
   dataTestId?: string | undefined;
   ouiaId?: string;
   ariaLabel: string | undefined;
   helperText: string | undefined;
-  validator: (value: string | undefined) => Boolean;
+  validator: (value: string | undefined) => boolean;
   value: string;
   placeholder?: string;
 }
 
+interface StateValidatedTextInputPropTypes extends TextInputProps {
+  dataTestId?: string | undefined;
+  ouiaId?: string;
+  stepId: string;
+  inputId: string;
```

Comment:
New component, but it requires `stepId` and `inputId` so I've decided just to copy the component to ease up the transition, but the end goal would be to get all fields validated through this component